src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -295,7 +299,23 @@ watch(
                 </div>
                 <!-- eslint-disable vue/no-v-html-->
                 <div class="small text-wrap text-center" v-html="t('share_disclaimer')"></div>
-                <!-- eslint-enable vue/no-v-html-->
+                <div
+                    v-if="hasAnyExternalDataLocal"
+                    class="d-flex flex-row gap-2 justify-content-center mt-2"
+                    data-cy="warn-share-local-file-container"
+                >
+                    <FontAwesomeIcon
+                        class="data-disclaimer-tooltip text-warning align-self-center"
+                        icon="circle-exclamation"
+                        data-cy="menu-external-disclaimer-icon-cloud"
+                    />
+                    <div
+                        class="small text-wrap text-center text-warning"
```

Comment:
I would keep the text the standard color, only have the icon in `text-warning` (we do not have other places where we color text yellow yet, I don't find that it looks particularly good contrast-wise)